libs/langchain-openai/src/azure/llms.ts

```diff
@@ -27,15 +29,8 @@ export class AzureOpenAI extends OpenAI {
         configuration?: ClientOptions & LegacyOpenAIInput;
       }
   ) {
-    // assume the base URL does not contain "openai" nor "deployments" prefix
-    let basePath = fields?.openAIBasePath ?? "";
```

Comment:
Actually, this is breaking, isn't it?

I don't think this is heavily used yet and there are no examples in our docs, so I'm ok with a change now.